effectsize_example.Rmd

```diff
@@ -34,23 +43,28 @@ In sum, an effect size report should include:
 - The type of uncertainty reported (e.g., 95% CI)
 - The units of the effect size if applicable, or the type of standardized effect size if it is a unitless effect size. 
 
+▸ Exemplar: [simple effect size](#exemplar_simple_effect_size) (specifically in the "Reporting simple effect size" sub-section)
 
+<a id="faq_simple_v_standardized"></a>
 
-## Should simple effect sizes or standardized effect sizes be reported? {#simple_v_standardized}
+## Should simple effect sizes or standardized effect sizes be reported?
```

Comment:
In my setup (RStudio on a mac) `{#xxx}` does *not* compile into `<a id="xxx"/>`.